.github/workflows/dapr-test.yml

```diff
@@ -397,9 +409,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target_os: [linux]
-        target_arch: [amd64, arm64]
```

Comment:
Same as above, is it possible to remove arm64 here directly?

Reply:
@ItalyPaleAle Please, follow up on these.

Reply:
Same as above. The net effect is the same, this is just more clear because windows was not in the matrix. Now it’s easier to tell what gets executed.